aspnetcore/tutorials/first-mvc-app/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Create a web app with ASP.NET Core MVC
 author: rick-anderson
-description: See the table of contents for an introduction to ASP.NET Core MVC.
+description: This series of tutorials explains the basics of building a web app with ASP.NET Core MVC.
```

Comment:
```suggestion
description:  Provides the table of contents for an introduction to ASP.NET Core MVC.
```

Reply:
And when I agreed to two files, this one doesn't count.

Reply:
It made sense to me to group add controller and add view at the same time rather than get started and controller.